src/MongoDB.Driver.Core/Core/WireProtocol/Messages/Encoders/BinaryEncoders/BinaryMessageEncoderFactory.cs

```diff
@@ -103,5 +109,11 @@ public IMessageEncoder GetUpdateMessageEncoder()
         {
             return new UpdateMessageBinaryEncoder(_stream, _encoderSettings);
         }
+
+        /// <inheritdoc />
+        public IMessageEncoder GetCompressedMessageEncoder(IMessageEncoderSelector originalEncoderSelector)
```

Comment:
Were you going to put these in alphabetical order?

If so check other instances also.